extensions/amp-sidebar/1.0/component.js

```diff
@@ -190,45 +191,51 @@ export function SidebarToolbar({
   children,
   ...rest
 }) {
-  const ref = useRef();
-  const [matches, setMatches] = useState(false);
-  const [targetEl, setTargetEl] = useState(null);
+  const ref = useRef(null);
+  const [target, setTarget] = useState(null);
 
   useEffect(() => {
-    const window = ref.current?.ownerDocument?.defaultView;
-    if (!window) {
+    const doc = ref.current?.ownerDocument;
+    if (!doc) {
       return;
     }
 
-    const mediaQueryList = window.matchMedia(mediaQueryProp);
-    const updateMatches = () => setMatches(mediaQueryList.matches);
-    mediaQueryList.addEventListener('change', updateMatches);
-    setMatches(mediaQueryList.matches);
-    return () => mediaQueryList.removeEventListener('change', updateMatches);
-  }, [mediaQueryProp]);
+    const selector = `#${escapeCssSelectorIdent(toolbarTarget)}`;
+    const newTarget = doc.querySelector(selector);
```

Comment:
Nit: This'll be faster
```suggestion
    const newTarget = doc.getElementById(toolbarTarget);
```

Reply:
Thanks, updated in the latest!